HadesScheduler/HadesOperator/cmd/main.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2025.
```

Comment:
Is this auto-generated? And is this really the licence of this code potion? Hades is licenced under MIT